axiom/cmd_txndata_transmit.c

```diff
@@ -106,6 +106,89 @@ static uint32_t nr_txndata_prepend_custom_events(nr_flatbuffer_t* fb,
   return events;
 }
 
+static uint32_t nr_txndata_prepend_log_events(nr_flatbuffer_t* fb,
+                                              const nrtxn_t* txn,
+                                              size_t log_event_limit) {
+  uint32_t* offsets;
+  uint32_t* offset;
+  uint32_t events;
+  size_t i;
+  size_t event_count;
+  nr_vector_t* events_vec;
+  nrbuf_t* buf;
+
+  const size_t event_size = sizeof(uint32_t);
+  const size_t event_align = sizeof(uint32_t);
+
+  event_count = nr_log_events_number_saved(txn->log_events);
+  if (0 == event_count) {
+    return 0;
+  } else if (event_count > log_event_limit) {
```

Comment:
This might be just defensive programming on my part to make sure there is sanity between how the txn->log_events was created and what is later being requested sent to the daemon.  And the desire to make sure if the limit is set to 0 then nothing is sent?